ssdc-rm-common-entity-model/src/main/java/uk/gov/ons/ssdc/common/model/entity/UacQidLink.java

```diff
@@ -37,4 +43,8 @@
   @Column(columnDefinition = "timestamp with time zone")
   @UpdateTimestamp
   private OffsetDateTime lastUpdatedAt;
+
+  @Type(type = "jsonb")
+  @Column(columnDefinition = "jsonb")
+  private Map<String, String> uacMetadata;
```

Comment:
Should be `Object` or `String` to allow future flexibility beyond flat key:value pairs of strings